### ISSUE TITLE
Add topics/tags to the project description

### ISSUE THREAD

**Issue by Aashishkebab** [rsanchez-wsu/RaiderPlanner#123]
Add some tags describing what the project does. Like so: 
<img width="342" alt="screenshot_20171017-224136" src="https://user-images.githubusercontent.com/25588487/31698622-60178bba-b38c-11e7-891b-ece0ab7af77e.png">

**Comment by NathanS1** [comment]
I will look into this.

**Comment by Aashishkebab** [comment]
@NathanS1 this was more directed towards Sanchez, as you may not have the authorization to change the project tags.

**Comment by NathanS1** [comment]
Right, I suggested tags that could be added now and in the future based on the current state of the project.
